docs/tutorial.md

```diff
@@ -0,0 +1,104 @@
+# Tutorial
+
+Example can be found in the [examples](https://github.com/PrismarineJS/mineflayer/tree/master/examples) folder.
+
+## Creating Simple plugin
+
+Here basic example on how to create plugin.
```

Comment:
we already have doc there https://github.com/PrismarineJS/mineflayer/blob/master/docs/CONTRIBUTING.md on how to create a plugin